doc/styles/config/vocabularies/ANSYS/accept.txt

```diff
@@ -53,7 +53,9 @@ CentOS7
 Chao
 ci
 container_layout
+contibuting

```

Comment:
All of these typos, did you track them in an issue? or ask for MAPDL team to fix them?

Can you have the library run a final step where we replace these?

Reply:
Great then! Thank you!

Reply:
Done with https://github.com/ansys/mapdl-cmd-conv/pull/164/commits/83fbfe0c680722f011d99ef75af4372de71e2424!